src/app/Http/Controllers/Company/Employee/EmployeeLogsController.php

```diff
@@ -47,27 +47,24 @@ public function index(Request $request, int $companyId, int $employeeId)
         }
 
         // logs
-        if(!$request->get('filterByLogType') || $request->get('filterByLogType') == "show_all_log_types") {
-            $logs = $employee->employeeLogs()->with('author')->paginate(100)->withQueryString();
-            $search = "show_all_log_types";
-        } else {
-            $logs = $employee->employeeLogs()->with('author')->where('action', $request->get('filterByLogType'))->paginate(100)->withQueryString();
-            $search = $request->get('filterByLogType');
+        $search = $request->input('filterByLogType', '');
```

Comment:
This logic is shortened by assuming that an empty query parameter means "show all of the logs"